specifyweb/frontend/js_src/lib/components/DataModel/specifyTable.ts

```diff
@@ -324,9 +324,10 @@ export class SpecifyTable<SCHEMA extends AnySchema = AnySchema> {
     if (typeof unparsedName !== 'string') throw new Error('Invalid field name');
 
     const splitName = unparsedName.toLowerCase().trim().split('.');
-    let fields = filterArray([
-      this.fields.find((field) => field.name.toLowerCase() === splitName[0]),
-    ]);
+    const exactMatch = this.fields.find(
```

Comment:
micro-optimization to avoid filterArray here
(I run a profiler and it showed a small slow down and garbage collector on this line, so decided to optimize it as it's a hot path)